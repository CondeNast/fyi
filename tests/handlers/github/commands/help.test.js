const helpEvent = require('../../../fixtures/help')
const models = require('../../../../models')

let app
let github

describe('Arch Bot', () => {
  beforeEach(() => {
    ({app, github} = require('../../../setup.js'))
  })

  describe('Help', () => {
    it('Posts Comment in Issue in Admin Repository', async () => {
      await app.receive(helpEvent)
      expect(github.issues.createComment).toMatchSnapshot()
    })
  })

  afterAll(async () => {
    await models.sequelize.close()
  })
})
