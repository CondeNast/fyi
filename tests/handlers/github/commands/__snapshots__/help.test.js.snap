// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arch Bot Help Posts Comment in Issue in Admin Repository 1`] = `
[MockFunction] {
  "calls": Array [
    Array [
      Object {
        "body": "Here are commands you can run:
  - \`/request [fyi name]\` to request for an FYI to be created
  - \`/skip\` to skip requests
  - \`/accept\` to accept submitted FYIs
  - \`/reject [comment]\` to reject submitted FYIs with optional comment
  - \`/close\` to close this issue
  - \`/remind\` to post a reminder on the request issue
",
        "number": 292,
        "owner": "choosenearme",
        "repo": "fyis",
      },
    ],
  ],
}
`;
